ivy_tests/test_ivy/test_frontends/test_mindspore/test_ops/test_function/test_mindspore_nn_func.py

```diff
@@ -77,3 +79,67 @@
 #         on_device=on_device,
 #         x=x[0],
 #     )
+
+# interpolate
+# @given(size_or_scale_factor=st.one_of(st.integers(min_value=1, max_value=10), st.tuples(st.integers(min_value=1, max_value=10)), st.lists(st.integers(min_value=1, max_value=10), min_size=3, max_size=3), st.none()))
```

Comment:
We should generally add these strategies inside `handle_frontend_test` rather than in a `given` decorator.

Also, if you're using this for the scale factor then it should be able to generate floats, rather than just integers (I think they should be separate though).